packages/viewport/src/index.ts

```diff
@@ -95,6 +96,7 @@ function createMediaQueryList( args?: QueryOption ): QueryItem {
 // See https://github.com/WordPress/gutenberg/blob/f1d0bd550f85f5fa4279a3fdb9a2b9c28a7544c6/packages/base-styles/_mixins.scss#L27
 const mediaQueryOptions: Record< string, QueryOption > = {
 	'<480px': { max: 480 },
+	'<600px': { max: 600 },
```

Comment:
@p-jackson we probably don't want to add another breakpoint here, but this one is actually the value from `break-small` that is coming from Gutenberg, and I believe that's what we want moving forward (that's why I see `>660px` being used with a "deprecated" symbol within CSS.)

Should we keep this here? Is there a better way of referencing this specific breakpoint?

Reply:
If we're going to move the **Continue** button from the main body into the toolbar I can't think of a way to do this that doesn't involve JavaScript unfortunately.

I think `useViewportMatch` might be better though, since it's already a Core thing and uses the standard GB break points.
[Like this example](https://github.com/Automattic/wp-calypso/blob/a5d5804c343c648deb0dcc473142e82db4862175/client/signup/steps/courses/index.tsx#L22)

Also what I like better is it consistently enforces either `<` or `>=`, because of course `<600` is not the exact opposite of `>600`. That confusion I think is creating this behaviour.


https://github.com/user-attachments/assets/97045a9a-1536-49ab-8925-d7b2a723ad79

It's probably not that big a deal. And switching to `useViewportMatch` might not fix it immediately. The stepper frameworks mobile toolbar might not be using the standard mixing breakpoint, it might be using the `break-small` variable direction, which means the logic could be a bit busted still.